cmd/git-sync/main.go

```diff
@@ -183,6 +189,36 @@ func envDuration(key string, def time.Duration) time.Duration {
 	return def
 }
 
+func sanitizeSpecificTime(t string) (h int, m int) {
+	if uniseg.GraphemeClusterCount(t) != 4 {
+		err := errors.New("error=")
+		log.Error(err, "--specific-time value not in military format, e.g. 1500 or 0130")
```

Comment:
compact  these line down with `log.Error(fmt.Errorf("--specific-time value not in military format, e.g. 1500 or 0130"))`

log.error will say error, no need for `error=`